lib/features/keyboard/Keyboard.js

```diff
@@ -22,50 +48,39 @@ import {
  * A default binding for the keyboard may be specified via the
  * `keyboard.bindTo` configuration option.
  *
- * @param {Config} config
+ * @param {Object} [config]
+ * @param {boolean} [config.invertY]
+ * @param {Number} [config.speed]
+ * @param {EventTarget} [config.bindTo]
  * @param {EventBus} eventBus
  * @param {EditorActions} editorActions
  */
 export default function Keyboard(config, eventBus, editorActions) {
   var self = this;
 
-  this._config = config || {};
+  this._config = assign({
+    speed: DEFAULT_SPEED,
+    invertY: false
```

Comment:
As mentioned above, the move canvas specifics belong to a move canvas keybinding if you ask me.

As an alternative we could make the detection implicit, i.e. apply it once we detect MacOS.

Reply:
The `speed` was already there, but it wasn't clearly expressed in the config declaraction. As the fallback, the implementation used a magic number. Otherwise, we could have a separate configuration for each key binding. I think it is a good idea.

Reply:
I think a better place to move the config is in [#376](https://github.com/bpmn-io/bpmn-js/issues/376). I'll change it, but in another PR.

Reply:
Same here. Speed shouldn't be part of the generic keyboard configuration. Other features might implement something that they want to do at a certain speed. If the configured speed doesn't work for two independent features they can't make use of it anyway.